pages/games/games.module.scss

```diff
@@ -1,10 +1,9 @@
 .container {
-  padding: 100px;
   text-align: center;
 
-  @media (max-width: 576px) {
-    padding: 40px 10px;
-  }
+  // @media (max-width: 576px) {
+  //   padding: 40px 10px;
+  // }
```

Comment:
is it supposed to be used in the future?

Reply:
these files will no longer be used. needs no fix

Reply:
If not in use any more, clear them out